packages/compass-collection/src/stores/index.ts

```diff
@@ -123,6 +171,7 @@ store.onActivated = (appRegistry: AppRegistry) => {
             sourcePipeline: metadata.sourcePipeline,
             query: metadata.query,
             aggregation: metadata.aggregation,
+            stats: metadata.stats,
```

Comment:
This `metadata` object doesn't have `stats` available

Reply:
Fixes opening new tabs with updated stats info. Hope I understood the data flow correctly. Pls, let me know if misused something.